wcivf/apps/feedback/forms.py

```diff
@@ -12,11 +12,12 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = Feedback
-        fields = ["found_useful", "comments", "source_url"]
+        fields = ["found_useful", "sources", "comments", "source_url"]
 
     found_useful = forms.ChoiceField(
         choices=FOUND_USEFUL_CHOICES,
         widget=forms.RadioSelect(attrs={"data-toggle": "button"}),
     )
+    sources = forms.CharField(widget=forms.HiddenInput())
```

Comment:
Not sure you want this to be a hidden field? In fact, as you have added `sources` to the `fields` list in the Meta, there is no need for this line at all - Django will render know to render the field as a TextInput as it is defined on the model as a `TextField`